OneCommunityGlobal/HighestGoodNetworkApp
# vitor fix delete badge when count is zero

fixes bug 11 from BADGES COMPONENT - (PRIORITY LOW) Jae: Select Featured changing count to 0 should delete a badge.

## Description
The Badges should disappear when the ‘count’ is set to zero, but they don’t.

## Main changes
1. Now, when the count is set to zero and the save button is clicked, the badges which the count is zero will be removed from the user's list of badges.

## How to test
1 - check into current branch
2 - do npm install and `npm run start:local` to run this PR locally
3 - log as admin user
4 - Go to dashboard → User Profile → Featured Badges → Assign Badges → Assign any quantity of  Badges to the user, then go to ‘Selected Featured’ and change the count to zero → Save.
5 - The badges which the count was zero when you saved should be removed from the user's profile.

## Example of how to test

https://user-images.githubusercontent.com/17365161/225116246-12dd09a3-1dbe-4eb2-9cb4-00518306aa9d.mp4
